pkg/chart/fake/chart.go

```diff
@@ -17,25 +17,19 @@ limitations under the License.
 package fake
 
 import (
-	"encoding/json"
-	"net/http"
-
+	appRepov1 "github.com/kubeapps/kubeapps/cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1"
 	chartUtils "github.com/kubeapps/kubeapps/pkg/chart"
-	"github.com/kubeapps/kubeapps/pkg/kube"
 	chart3 "helm.sh/helm/v3/pkg/chart"
+	corev1 "k8s.io/api/core/v1"
 	chart2 "k8s.io/helm/pkg/proto/hapi/chart"
 	"sigs.k8s.io/yaml"
 )
 
-type FakeChart struct{}
-
-func (f *FakeChart) ParseDetails(data []byte) (*chartUtils.Details, error) {
-	details := &chartUtils.Details{}
-	err := json.Unmarshal(data, details)
-	return details, err
-}
+// Chart implements Resolver inteface
```

Comment:
Note, a while back we noted that `Chart` was confusing for the implementation and had renamed in the non-fake module to `Client` (since what was called `Chart` was really a chart client). We should probably rename it here too so that...